client/css/editmode.css

```diff
@@ -30,9 +30,16 @@
   cursor: pointer;
 }
 
+#editJSONoverlay {
+  padding: 10px;
+}
+
+#editJSONoverlay h1 {
+  margin: 0;
+}
+
 #editJSONoverlay textarea {
-    height: 100%;
-    min-height: 100%;
+  height: 100%;
```

Comment:
.....

we reintroduced this yesterday 😑

we need it for mobile. I'll take a look at this PR

Reply:
Did you test it? I thought it might be enough without it now.

Reply:
I just tested on my phone and I think it's good enough. The problem with the `min-height` is that the buttons are off screen and if the text is long enough you get two nested scrollbars. So if it's good enough without it, I would prefer that.

Reply:
Before ![Screenshot_20211125-101032.png](https://user-images.githubusercontent.com/73476656/143447760-c286ead3-9eae-4131-8da5-3ecb2a253599.png)

After ![Screenshot_20211125-101155.png](https://user-images.githubusercontent.com/73476656/143447781-3fe89f73-26ae-4516-a3c2-46916bb0a9df.png)

Reply:
Holy shit, are you blind? :smile:

We should at least only do that when the viewport isn't high enough. It feels so broken to have to scroll to the buttons on a 4k screen.